Libraries/src/Amazon.Lambda.RuntimeSupport/ExceptionHandling/JsonExceptionWriterHelpers.cs

```diff
@@ -13,7 +13,7 @@
  * permissions and limitations under the License.
  */
 
-using System.Text;
+using System.Text.Json;
```

Comment:
It seems like in this [commit](https://github.com/aws/aws-lambda-dotnet/commit/b19394e0f77cd7cc17e4826e8fbe7bcf9633ab40), we migrated from LitJSON to System.Text.Json and left this method as is. I am not sure why though.